strategy/measuring-impact.md

```diff
@@ -0,0 +1,148 @@
+
+Scientists expect to benchmark datasets by processing and analysing them with computational methods within or alongside the DCP and comparing them to results from other methods.
+
+#### Indicators for Success
+
+| Specific aim | Item | Indicator |
+| -----------: | ---- | :-------- |
+| 1 | a. | DCP technology reports are highly accessed by a variety of users |
+|   | b. | DCP technology reports receive positive feedback from the wider single cell community |
+| 2 | a. | The number of datasets that are generated and deposited in the DCP, but that cannot be analysed by DCP pipelines, is very low |
+| 3 | a. | Scientists access data processed by core DCP pipelines more frequently than data processed by community-contributed pipelines |
+| 4 | a. | Processed data from community-contributed pipelines is heavily accessed |
+
+
+### Aim 5
+
+**Create standards for the community to use to describe single cell experimental designs, including assay types, data and metadata.**
+
+#### Value Assumptions
+
+Scientists would like the DCP to provide guidance on how to describe single cell experiment designs and would be prepared to invest time in accurately providing this information if it these standards are accepted by the community.
+
+Scientists want the DCP to drive convergence around a small number of methods, and expect the DCP to support this by facilitating direct comparison of  the quality of results acquired from different experiment designs.
+
+Scientists care about a core set of relatively simple fields to characterise cell types, but it is currently unclear what these are across the Human Cell Atlas (e.g. all tissues or all sampling techniques).
+
+For this set of fields, scientists would like data to be very well described, with detailed metadata and annotations to a standard set of ontologies.
+
+Scientists would like DCP data to be consistently described and interoperable with all cellular resolution data (both inside and outside the platform).
+
+#### Indicators for Success
+
+| Specific aim | Item | Indicator |
+| -----------: | ---- | :-------- |
+| 1 | a. | Most commonly used experimental designs in the DCP correlate to higher QC results |
+| 2 | a. | Summary tables of DCP data volume, aggregated by key metadata, e.g. which assays are applied to which tissues, are highly accessed. |
+| 3 | a. | Captured biological metadata (biological materials and experiment designs) are frequently used to characterise cell types |
+|   | b. | Captured computation metadata (analysis methods, QC and file formats) are used when determining which data to use in the characterisation of cell types |
+| 4 | a. | Portals combine and visualise data from within the DCP and from at least one other source |
+|   | b. | Publications that perform integrative analyses of data from within the DCP and from at least one other source are highly cited |
```

Comment:
Seems like having another project use our metadata schema would indicate success as well?

Reply:
@briandoconnor Thanks for the feedback - there's some excellent discussion topics in here. I'd like to address them, but I also recognise a desire to get _"strategy 1.0"_ published.

Given that the strategy has always been envisioned as a living document that should be modified as we learn more about what our users need, would you object if I held off on these comments for now? I'd be very happy to open a subsequent PR to tackle your insights if that works for you?